.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "era-compiler-tester"]
 	path = era-compiler-tester
 	url = https://github.com/lambdaclass/era-compiler-tester.git
-	branch = zksync-era-tests
+	branch = decommited-hashes
```

Comment:
shouldn't we keep the submodule branch as is? It would be helpful for others if we link the corresponding PR in `era-compiler-tester` in the description, so they know they should be seen in tandem.